Documentation/content/docs/intro_vtk_as_es6_dependency.md

```diff
@@ -1,243 +1,76 @@
 title: Using vtk.js as an ES6 dependency
 ---
 
-This guide illustrates how to build an application using vtk.js as a dependency using a modern toolsets such as Webpack, NPM.
+This guide illustrates how to consume vtk.js as an ES dependency.
 
-## Creation of the Project structure
+## Starting a new vtk.js project
 
-```sh
-$ mkdir MyWebProject
-$ cd MyWebProject
-$ npm init
-
-  This utility will walk you through creating a package.json file.
-  It only covers the most common items, and tries to guess sensible defaults.
-
-  See `npm help json` for definitive documentation on these fields
-  and exactly what they do.
-
-  Use `npm install <pkg> --save` afterwards to install a package and
-  save it as a dependency in the package.json file.
-
-  Press ^C at any time to quit.
-  name: (MyWebProject) your-project-name
-  version: (1.0.0) 0.0.1
-  description: vtk.js application
-  entry point: (index.js) src/index.js
-  test command:
-  git repository:
-  keywords: Web visualization using VTK
-  author:
-  license: (ISC) BSD-2-Clause
-  About to write to /.../MyWebProject/package.json:
-
-  {
-    "name": "your-project-name",
-    "version": "0.0.1",
-    "description": "vtk.js application",
-    "main": "src/index.js",
-    "scripts": {
-      "test": "echo \"Error: no test specified\" && exit 1"
-    },
-    "keywords": [
-      "Web",
-      "visualization",
-      "using",
-      "VTK"
-    ],
-    "author": "",
-    "license": "BSD-2-Clause"
-  }
+There are several ways to start a new vtk.js project. Below lists a few ways you can begin using vtk.js with your favorite project build tool.
 
+- [vtk.js from scratch (with webpack)](./vtk_vanilla.html)
+- [vtk.js with React](./vtk_react.html)
+- [vtk.js with vue](./vtk_vue.html)
 
-  Is this ok? (yes)
-```
+## Adding vtk.js to an existing project
 
-Then install and save your dependencies
+You can add vtk.js to an existing project as follows:
 
 ```sh
-$ npm install vtk.js --save
-$ npm install kw-web-suite --save-dev
+$ npm install --save @kitware/vtk.js
```

Comment:
Is the save automatic now?

Reply:
I think save is automatic, but I'll keep the flag here. I don't know which versions of npm have it.